mentoring/answer.py

```diff
@@ -24,39 +21,146 @@
 # Imports ###########################################################
 
 import logging
-
 from lazy import lazy
 
-from xblock.fragment import Fragment
-
-from .light_children import LightChild, Boolean, Scope, String
 from .models import Answer
-from .utils import render_template
+
+from xblock.core import XBlock
+from xblock.fields import Scope, Float, Integer, String
+from xblock.fragment import Fragment
+from xblock.validation import ValidationMessage
+from xblockutils.resources import ResourceLoader
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+from .step import StepMixin
+import uuid
 
 
 # Globals ###########################################################
 
 log = logging.getLogger(__name__)
+loader = ResourceLoader(__name__)
+
 
+# Make '_' a no-op so we can scrape strings
+def _(text):
+    return text
```

Comment:
Braden - the string extractions look great. Have you tested to see if the
block actually displays the dummy language translations when installed in
edx-platform? Making sure the round-trip happens correctly is the final
step. If the dummy translations display, then that makes it easy to add
translations (which may happen if, say, a course running in another
language wants to use the block and wishes to provide translations).

On Fri, Mar 20, 2015 at 1:32 AM, Braden MacDonald <notifications@github.com>
wrote:

> In mentoring/answer.py
> <https://github.com/open-craft/xblock-mentoring/pull/4#discussion_r26819606>
> :
>
> >
> > +# Make '_' a no-op so we can scrape strings
> > +def _(text):
> > +    return text
>
> @sarina <https://github.com/sarina> re: extracting strings - I took a
> stab at that with a4f8269
> <https://github.com/open-craft/xblock-mentoring/commit/a4f826981bfeafab740c2445507cf9405d4d9c0c>
> - anything I should change? I just did English and a dummy translation for
> now, since I don't know of any plans to actuate translate this into any
> particular languages - but let me know if I'm missing something.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/open-craft/xblock-mentoring/pull/4/files#r26819606>.
>